### ISSUE TITLE
Randomly lost the ability to use previously working firmware + games

### ISSUE THREAD

**Issue by shadowofdarkness** [mrehkopf/sd2snes#172]
I have a strange problem that just started today. I went to turn my Fxpak Pro (with the latest beta firmware) on and it came up to a scrambled screen when it was working last time I turned it on. I checked the memory card for errors and found some so I reformatted it and put fresh files on from the original release zip and got the same problem of a scrambled menu, although the Near memorial screen was fine.

After some testing on 3 different SD cards I find them all to have the same issue. Downgrading to the last stable release works and I can start games with the exception of the only 2 games I put real time into playing which can't be a coincidence any other ROM quickly tested works.

The non working games are a Super Mario World romhack and SMRPG I have tested the files hashes against the originals and they are the same so it shouldn't be file corruption. They just go to a black screen when started.

How can this happen, other then the memory card does anything get saved to the cartridge that could be corrupted I have only had it for a few months with only around 15 hours playtime and it always worked fine before.

Considering it only affects the one firmware version which before today was the only one ever used on it and it also only affects the games I was playing. I don't think it could be bad capacitors or bad hardware.

No old files were used when setting up the 3 cards and all got setup separate. So it has to be a problem with something else.

Edit Both the official burn in test rom and the Faxpak Pro diagnostic firmware report no problems with the system.

**Comment by shadowofdarkness** [comment]
I may have narrowed down the problem. I found out that the specific SMW rom hack I was playing actually required the SA1 enhancement chip that the plain game didn't and that same chip was used by SMRPG. I spot tested a game from every other chip I could and they worked but just SA1 seemed to be a problem.
Of the SA1 games I tested I found a Kirby game that also just starts to a black screen but then the other 2 I tested did work so it is not completely gone.

**Comment by shadowofdarkness** [comment]
Turns out my problem is the console not the Fxpak Pro. It took a bit but I found a second console to borrow for a test and it had no problems both the latest firmware and all 3 problem games worked again.

I'm hoping it is just a power issue and new caps will fix it I just find it strange that the newest firmware release has a problem but not older versions. I could understand the games as maybe SA1 uses more power then other enhancement chips.
